tests/test_broadcast.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 
 from broadcaster import Broadcast
+from broadcaster._backends.memory import MemoryBackend
```

Comment:
🐼 Can we test against public API?

Reply:
Ah understood.

Given the intent of this PR perhaps the test could be against `broadcaster.BroadcastBackend`?

Reply:
`broadcaster.BroadcastBackend` is an abstract class. Do you mean extending it for this particular case?

Reply:
Yep.
For `httpx` we have a policy of only testing against the public API, which seems a good pattern.
We could be following this here also, right?